bot/src/infrastructure/http/ZkillClient.ts

```
@@ -1,47 +1,96 @@
+    labels?: string[];
+  };
+  [key: string]: any; // Allow additional fields
+}
 
 export class ZkillClient {
   private readonly client: UnifiedESIClient;
-  private lastRequestTime: number = 0;
-  private readonly rateLimitMs: number = 1000; // 1 request per second
+  private readonly rateLimiter: RateLimiter;
 
   constructor(baseUrl: string = "https://zkillboard.com/api") {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)